tests/common/plugins/conditional_mark/tests_mark_conditions.yaml

```diff
@@ -339,6 +339,26 @@ dhcp_relay/test_dhcp_relay.py::test_dhcp_relay_unicast_mac:
     conditions:
       - "'dualtor' in topo_name and release in ['201811', '201911']"
 
+dhcp_relay/test_dhcp_relay_stress.py::test_dhcp_relay_stress:
+  skip:
+    reason: "1. Need to skip for platform armhf-nokia_ixs7215_52x-r0 due to buffer issues
+             2. Need to skip for KVM due to low performance"
+    conditions:
+      - "platform in ['armhf-nokia_ixs7215_52x-r0']"
```

Comment:
Default logic is `AND`, could you refer to `conditions_logical_operator: OR` in this file to check?